app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -599,8 +599,8 @@ def hide_button?(id)
       type = get_vmdb_config.fetch_path(:server, :remote_console_type)
       return type != 'MKS' || !@record.console_supported?(type)
     when "vm_vnc_console"
-      type = get_vmdb_config.fetch_path(:server, :remote_console_type)
-      return type != 'VNC' || !@record.console_supported?(type)
+      # :remote_console_type is only useful for vmware consoles
+      return !@record.console_supported?('vnc')
```

Comment:
The goal of my PR is to have the VMware vms respect the console setting in the Configuration screen and have only that console button available to the user.  The Configuration screen allows VMware consoles to be set to MKS, VMRC or VNC. 

If that setting is not a valid configuration for VMware vms, VNC should be removed from the VMware Console Support dropdown and my suggested changes ignored.

If it is a valid configuration option, I believe my suggested change above should be implemented so that we respect the selection of the admin rather than ignoring it. 

If neither of those options is seen as the way forward, we need to revert the names of the buttons so that we do not have two 'VM Console' buttons in the Access menu on every VMware vm.